client/state/themes/utils.js

```diff
@@ -4,9 +4,85 @@
  * External dependencies
  */
 import startsWith from 'lodash/startsWith';
+import { omit, omitBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_THEME_QUERY } from './constants';
+
+/**
+ * Constants
+ */
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
 
 export const oldShowcaseUrl = '//wordpress.com/themes/';
 
+/**
+ * Utility
+ */
+
+/**
+ * Returns a normalized themes query, excluding any values which match the
+ * default theme query.
+ *
+ * @param  {Object} query Themes query
+ * @return {Object}       Normalized themes query
+ */
+export function getNormalizedThemesQuery( query ) {
```

Comment:
Tricky. A function name that uses a verb (to me, at least) kind of suggests that it transforms an argument, or triggers an action. It's true that the `get` suggests it might be a selector, but I think it's just for lack of synonyms that both use `get`.

Anyway, this is once again code stolen from the `posts` state, so I'd keep the `get`, if only for consistency's sake.